components/modal/index.css

```
@@ -13,15 +13,20 @@ governing permissions and limitations under the License.
 @import "../overlay/index.css";
 
 .spectrum-Modal {
-	--spectrum-dialog-confirm-vertical-offset: -2vh;
-
 	/* Bug: this must be 0ms, not 0 */
-	--spectrum-dialog-confirm-exit-animation-delay: 0ms;
-
-	/* Distance between top and bottom of dialog and edge of window for fullscreen dialog */
-	--spectrum-dialog-fullscreen-margin: 32px;
-	--spectrum-dialog-max-height: 90vh;
-	--spectrum-dialog-max-width: 90%;
+	--spectrum-modal-confirm-exit-animation-delay: 0ms;
+
+	/* Distance between top and bottom of modal and edge of window for fullscreen modal */
+	--spectrum-modal-fullscreen-margin: 32px;
+	--spectrum-modal-max-height: 90vh;
+	--spectrum-modal-max-width: 90%;
+
+	--spectrum-modal-background-color: var(--spectrum-gray-100);
+	--spectrum-modal-confirm-border-radius: var(--spectrum-corner-radius-100);
+	--spectrum-modal-confirm-exit-animation-duration: 130ms;
```

Could you change this to:

```suggestion
	--spectrum-modal-confirm-exit-animation-duration: var(--spectrum-animation-duration-100);
```